README.md

```diff
@@ -55,7 +55,7 @@ These can be installed with `dpkg -i summon_v*.deb` and
 
 ### Auto Install
 
-**Note** Check the release notes and select an appropriate release to ensure support for your version of Conjur.
+**Note** Check the [release notes](https://github.com/cyberark/summon/releases) and select an appropriate release to ensure support for your version of Conjur.
```

Comment:
This line is longer than 80 chars, so CodeClimate will complain. This can be split into 3 lines (the rendered README.md will still look the same).

Reply:
ok, will update.